x-pack/plugins/enterprise_search/public/applications/vector_search/components/dev_tools_console_code_block/dev_tools_console_code_block.tsx

```diff
@@ -60,7 +60,13 @@ export const DevToolsConsoleCodeBlock: React.FC<DevToolsConsoleCodeBlockProps> =
             <EuiFlexItem grow={false}>
               <EuiCopy textToCopy={children}>
                 {(copy) => (
-                  <EuiButtonEmpty color="text" iconType="copyClipboard" size="s" onClick={copy}>
+                  <EuiButtonEmpty
+                    data-test-subj="enterpriseSearchDevToolsConsoleCodeBlockCopyButton"
```

Comment:
@elastic/enterprise-search-frontend When I merged the main, I had a lint issue locally git commit hook that was automatically solved by adding this `data-test-subj`. 
Please let me know if you prefer not to have this change and I will remove this update.